ci/github/phpunit/ibm_db2.xml

```diff
@@ -16,6 +16,12 @@
         <var name="db_user" value="db2inst1"/>
         <var name="db_password" value="Doctrine2018"/>
         <var name="db_dbname" value="doctrine"/>
+
+        <var name="tmpdb_driver" value="ibm_db2"/>
+        <var name="tmpdb_host" value="127.0.0.1"/>
+        <var name="tmpdb_user" value="db2inst1"/>
+        <var name="tmpdb_password" value="Doctrine2018"/>
+        <var name="tmpdb_dbname" value="doctrine"/>
```

Comment:
What do we need those variables for?

Reply:
We need these variables to build the privileged connection in the test suite. The `connect()` method in DB2 requires the "dbname" option to be defined:

https://github.com/doctrine/dbal/blob/bf27755bfe9f6a04f5fa7adb77ff30e204723ef3/src/Connection.php#L446

The "userland" connection strips out that option:

https://github.com/doctrine/dbal/blob/bf27755bfe9f6a04f5fa7adb77ff30e204723ef3/tests/TestUtil.php#L165-L175

Reply:
Okay, but how does this relate to the introduction of the new platform class? I mean, we already have a working CI. Why is this set of configuration variables suddenly necessary?

Reply:
Because now the version detection is required (we are not passing the `serverVersion` option for the connection) to choose the right platform.

Reply:
Oh, I see. Thanks for clarifying.